build.rs

```diff
@@ -7,5 +7,11 @@ compile_error!("You have the `http` or `gateway` feature enabled, \
     - `native_tls_backend` uses SChannel on Windows, Secure Transport on macOS, \
     and OpenSSL on other platforms.\n\
     If you are unsure, go with `rustls_backend`.");
+#[cfg(all(feature = "voice", not(any(feature = "secretbox", feature = "sodiumoxide"))))]
```

Comment:
Please add an empty line between both `compile_error`-macros, also make sure to refer to `*_backend` instead of the actual crate names.